docs/deployment/authorization-and-authentication/Authenticating-Users-via-SAML.md

```diff
@@ -127,68 +127,28 @@ Within application.properties, make sure that:
 
 Then, modify the section labeled `authentication`. See SAML parameters shown in example below:
 
-    saml.sp.metadata.entityid=cbioportal
-    saml.sp.metadata.wantassertionsigned=true
-    saml.idp.metadata.location=classpath:/onelogin_metadata_620035.xml
-    saml.idp.metadata.entityid=https://app.onelogin.com/saml/metadata/620035
-    saml.keystore.location=classpath:/samlKeystore.jks
-    saml.keystore.password=apollo1
-    saml.keystore.private-key.key=secure-key
-    saml.keystore.private-key.password=apollo2
-    saml.keystore.default-key=secure-key
```

Comment:
It seems we still have some outdated keystore contents here: https://github.com/cBioPortal/cbioportal/blob/master/docs/deployment/authorization-and-authentication/Authenticating-Users-via-SAML.md?plain=1#L88-L110, maybe we can replace these with this: https://github.com/cBioPortal/cbioportal/blob/85fd3d9c866858ec412079b3ede3b68583a2d3ba/docs/deployment/customization/security.properties-Reference.md?plain=1#L69-L72